src/Containers/Microsoft.NET.Build.Containers/ContainerHttpException.cs

```diff
@@ -6,12 +6,10 @@ namespace Microsoft.NET.Build.Containers;
 internal sealed class ContainerHttpException : Exception
 {
     private const string ErrorPrefix = "Containerize: error CONTAINER004:";
-    string? jsonResponse;
     string? uri;
-    public ContainerHttpException(string message, string? targetUri, string? jsonResp)
-            : base($"{ErrorPrefix} {message}\nURI: {targetUri ?? "Unknown"}\nJson Response: {jsonResp ?? "None."}")
+    public ContainerHttpException(string message, string? targetUri)
```

Comment:
I don't think we should remove the `jsonResp` here - registries often have json payload describing what went wrong with the upload that can be helpful.

Reply:
Ah, I see what we started doing here - we started just logging the full details and hiding the raw responses from the user. See the `LogHttpResponseAsync` extension method used in `DefaultBlobUploadOperations` for an example. In that case, please do the following:
* keep your code the same
* in the DefaultManifestOperations call that throws the ContainerHttpException, could you call the `LogHttpResponseAsync` extension to log the data to the binlog?

That would keep the analysis data around, and clean up the UX for users.

Reply:
> could you call the LogHttpResponseAsync extension to log the data to the binlog?

It's already there :)

Reply:
Good grief, I am bad at comprehension this morning. Thank you for walking me through that. In that case this PR is 100% perfect, no notes.